dev-tools/global.d.ts

```diff
@@ -10,12 +10,13 @@ import * as RoomlogsType from './../roomlogs'
 import * as LadderStoreType from './../ladders-remote'
 import * as LaddersType from './../ladders'
 import * as UsersType from './../users'
+import * as PunishmentsType from './../punishments'
 
 declare global {
 	// modules
 	const Dex: typeof DexType
 	const Chat: any
```

Comment:
Can you make another pullreq importing Chat here?

You actually have a lot of bugs in Chat because you forgot to actually export it globally.

Reply:
Yeah I can do that in another pr, my bad